.gitignore

```diff
@@ -21,3 +21,5 @@ yarn-error.log*
 *.sln
 *.sw*
 
+# jest testing framework
+/coverage
```

Comment:
maybe it is worth also to add package-json file here?

Reply:
`package-lock.json`, `yarn.lock` should be always on repo. The only thing is that you gotta decide which one you use.